src/pages/DogProfileSettingsPage/DogProfileSettingsPage.tsx

```diff
@@ -76,7 +76,7 @@ const DogProfileSettingsPage: React.FC = (): React.ReactElement => {
     <Box sx={{ width: 300 }}>
       <form onSubmit={handleSubmit(onSubmit)}>
         <Stack spacing={4}>
-          <Typography variant="h1">Create Dog</Typography>
+          <Typography variant="h1" data-testid="dog-settings-header">Create Dog</Typography>
```

Comment:
I never considered adding a data label for testing this way. Looks like a good idea!